Sources/Extensions/UIKit/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIStackViewExtensions.swift
+//  SwifterSwift-iOS
+//
+//  Created by Benjamin Meyer on 2/18/18.
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+    
+// MARK: - Initializers
+@available(iOS 9.0, *)
+public extension UIStackView {
+    /// SwifterSwift: Initialize an UIStackView with an array of UIView and common parameters.
+    ///
+    ///        let stackView = UIStackView(views: [UIView(), UIView()], orientation: .vertical)
+    ///
+    /// - Parameter views: The UIViews to add to the stack.
```

Comment:
Please use Xcode to create docstrings template, [more](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md#power-tip).